.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,19 @@
+name: nv-ingest pre-commit
+
+on:
+    pull_request:
+    push:
+      branches: [main]
+
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
```

Comment:
We should remove this line; [it's vulnerable to script injection](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#example-of-a-script-injection-attack) if someone opens a PR and names their repo in a malicious manner.

Reply:
(Any user modifiable field is at risk of this, repo is probably the least likely but still something we shouldn't open ourselves to.)

Reply:
Agreed. Honestly I just copied this example from a silly little boilerplate example in their docs. I'll make some meaningful changes here shortly.